specs/commandline/list.md

```diff
@@ -2,12 +2,23 @@
 
 ## Description
 
-Use `notation list` to list all the signatures of the signed artifact.
+Use `notation list` to list all the signatures associated with signed artifact.
+
+`Tags` are mutable, but `Digests` uniquely and immutably identify an artifact. If a tag is used to identify a signed artifact, notation resolves the tag to the `digest` first.
+
+Upon successful execution, both the digest of the signed artifact and the digests of signatures associated with signed artifact are printed out as following:
+
+```shell
+<registry>/<repository>@<digest>
+└── application/vnd.cncf.notary.v2.signature
+    ├──<digest_of_signature>
+    └──<digest_of_signature>
```

Comment:
nit: Upon successful signature evaluation, we will short-circuit evaluation when we find verification succeeds for some signature